src/main/webapp/js/reducers/auth-reducer.js

```diff
@@ -29,6 +35,18 @@ export default (state = initialState, action) => {
       return { ...state, permissions: action.payload };
     case AUTH_ERROR:
       return { ...state, error: action.payload, authenticated: false };
+    case STORE_LOGIN: {
+      const {
+        username, hash, token, refreshToken,
+      } = action.payload;
+      return {
+        ...state,
+        savedLogins: {
+          ...state.savedLogins,
+          [username]: { hash, token, refreshToken },
```

Comment:
token is not needed here, save the permissions instead
I'm not sure about the refresh token, remove it for now it can be added later when it will be needed